boards/BMS/Inc/Io/Io_Adc.h

```diff
@@ -19,6 +19,12 @@ uint16_t *Io_Adc_GetRawAdc2Values(void);
  */
 float Io_Adc_GetAdc1Channel3Voltage(void);
 
+/**
+ * Get the voltage measured at ADC1 channel 4
+ * @return The voltage measured at ADC1 channel 4, in volts
+ */
+float Io_Adc_GetAdc1Channel4Voltage(void);
```

Comment:
we shouldnt need this for differential ADC correct?

Reply:
no we dont. removed it